types/algoliasearch/index.d.ts

```diff
@@ -1845,6 +1883,64 @@ declare namespace algoliasearch {
   interface MultiResponse {
     results: Response[];
   }
+
+  type QueryLanguage =
+    | 'af'
```

Comment:
How often does it happen that the number of supported query languages change? If it's a very rare occasion then I'd be fine with hardcoding it, otherwise I'm not too sure.

Reply:
Yes, language change would be very rare, I'll look again at this PR (and what failed the build) at a later stage